frontend/src/app/dashboard/dashboard.component.scss

```diff
@@ -213,4 +213,91 @@
 
 .terms-of-service {
   margin-top: 1rem;
+}
+
+.small-bar {
+  height: 8px;
+  top: -4px;
+}
+
+.difficulty-adjustment-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  .shared-block {
```

Comment:
@MiguelMedeiros it looks like you're adding a span with this class to components that don't need it (footer, transaction and mempool-blocks).

Reply:
I talked to @softsimon about it. We decided to add the html tag (span) with a new class to handle css separately in each component. But to work the translation correctly I needed to change all the components that use the translation of "shared.blocks" and "shared.block". So we can use the same strings without creating new ones.